flixel/tweens/FlxTween.hx

```diff
@@ -406,8 +410,61 @@ class FlxTween implements IFlxDestroyable
 		onUpdate = null;
 		onComplete = null;
 		ease = null;
+		_thens = null;
 	}
-
+	
+	/**
+	 * After this tween has finished, do this tween next
+	 * @param	method
```

Comment:
Might as well delete these empty `@param` tags, especially considering they're out of sync with the method signature.